html/cross-origin-embedder-policy/require-corp.https.html

```diff
@@ -0,0 +1,135 @@
+<!doctype html>
+<meta name="timeout" content="long">
+<title>Cross-Origin-Embedder-Policy header and nested navigable resource without such header</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script src="/common/get-host-info.sub.js"></script>
+<script src="/common/utils.js"></script> <!-- Use token() to allow running tests in parallel -->
+<div id=log></div>
+<script>
+async_test(t => {
+  const frame = document.createElement("iframe");
+  t.step_timeout(() => {
+    // Make sure the iframe didn't load.
+    assert_equals(frame.contentDocument, null);
+    t.done();
+  }, 500);
+  frame.src = "/common/blank.html";
```

Comment:
Our implementation fires a 'load' event while contentDocument is null. Does it match your expectation? If so, can we rely on that event instead of timer? Otherwise, can we use assert_unreached? Should we fire an 'error' event?

Reply:
For error pages it's not really well-defined what should happen. There's some open issues around that. I could point to those here I suppose for clarity.

Reply:
Thanks for the clarification.